pkgs/dart_services/lib/src/common_server.dart

```diff
@@ -38,12 +39,12 @@ class CommonServerImpl {
   late Analyzer analyzer;
   late Compiler compiler;
   final ai = GenerativeAI();
+  final genui = GenUi();
```

Comment:
```suggestion
  final GenUi genui = GenUi();
```

(and the same for the line above - existing code from a recent PR)

Reply:
Applied, but this should be regulated by lints, so that we do not spend time on code review.

Reply:
agree; I am a little surprised that this wasn't caught by anything.